actions/release_component_test.go

```diff
@@ -0,0 +1,34 @@
+package actions
+
+import "testing"
```

Comment:
I do see where a test could be added w/o much additional effort validating the json intending to be posted to the GH API endpoint (release name, formatting, etc., etc.)  

An example that might be helpful from a while back testing a different endpoint can be found [here](https://github.com/deis/deisrel/blob/9340bfe4ae3a47885f93475f72d223e42dcfca4a/git/create_branches_test.go)